src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanC100Service.java

```diff
@@ -59,12 +115,32 @@ public BulkScanTransformationResponse transform(BulkScanTransformationRequest bu
             Collectors.toMap(OcrDataField::getName, OcrDataField::getValue));
 
         Map<String, Object> populatedMap = (Map<String, Object>) BulkScanTransformHelper
-            .transformToCaseData(transformConfigManager.getTransformationConfig(FormType.C100)
-                    .getCaseDataFields(), inputFieldsMap);
+            .transformToCaseData(new HashMap<>(transformConfigManager.getTransformationConfig(FormType.C100)
+                    .getCaseDataFields()), inputFieldsMap);
+
+        populatedMap.put(CHILD_LIVE_WITH_KEY, getChildLiveWith(inputFieldsMap));
+
+        Map<String, String> caseTypeAndEventId =
+            transformConfigManager.getTransformationConfig(FormType.C100).getCaseFields();
```

Comment:
pls check scan documents url not managed this logic and add logic from other services

Reply:
yes based on other service only it was done